src/languages/es.ts

```diff
@@ -3234,6 +3234,16 @@ export default {
         },
         copyReferralLink: 'Copiar enlace de invitación',
     },
+    onboardingBottomMessage: {
+        [CONST.INTRO_CHOICES.MANAGE_TEAM]: {
+            phrase1: 'Chatea con tu especialista asignado en ',
+            phrase2: ' para obtener ayuda',
+        },
+        default: {
+            phrase1: 'Envía un email a ',
+            phrase2: ' para obtener ayuda con la configuración',
+        },
```

Comment:
I would check the style does not go crazy in Spanish because I think the translations is slightly longer